src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.StrategyOperation;
+
+public class BalanceOperation implements StrategyOperation {
+
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
```

Comment:
давай ініціалізовувати змінну через конструктор
```suggestion
    private StorageDao storageDao;
```
те саме з іншими класами